lib/cc/services/github_pull_requests.rb

```diff
@@ -40,30 +40,34 @@ def receive_test
 
   def receive_pull_request
```

Comment:
This method is getting pretty long and hard to read. Ideas on breaking it down? De-compose the method?